docker/docker.go

```diff
@@ -111,7 +111,7 @@ func main() {
 			*flTls = true
 			cert, err := tls.LoadX509KeyPair(*flCert, *flKey)
 			if err != nil {
-				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted?", err)
+				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted", err)
```

Comment:
Thx to duglin, I have go the right meaning now, and shall we change the message to express more clearly.
such as,"Couldn't load X509 key pair: %s.The Key may be encrypted."